packages/@lwc/engine/src/framework/api.ts

```diff
@@ -116,18 +114,12 @@ const TextHook: Hooks<VText> = {
 // which breaks some invariants. For that reason, we have the following for any
 // Custom Element that is inserted via a template.
 const ElementHook: Hooks<VElement> = {
-    create: vnode => {
-        const { data, sel, clonedElement } = vnode;
+    create: (vnode: VElement) => {
+        const { data, sel } = vnode;
         const { ns } = data;
-        // TODO [#1364]: supporting the ability to inject a cloned StyleElement via a vnode this is
-        // used for style tags for native shadow
-        if (isUndefined(clonedElement)) {
```

Comment:
I don't think we can remove this logic. This is about adapted stylesheets, which is not ready yet.

Reply:
The original commit where this was removed was this https://github.com/salesforce/lwc/pull/1431/commits/e990f360e069f4e4862bcb6696a1e91809579c2a. May be you recall by looking at that commit?

Reply:
no no, as I said in another comment, I understand now... this is not longer needed because we don't generate a vnode for the style anymore, we just add the style element into the shadowroot as the first element, and we get away with that. not need to have the clonedElement anymore.